modules/analytics-v2/src/views/full/index.tsx

```diff
@@ -85,38 +97,84 @@ export default class Analytics extends React.Component<{ bp: any }> {
+        <DateRangeInput
+          closeOnSelection={true}
+          formatDate={date => moment(date).format('MM-DD-YYYY')}
+          maxDate={new Date()}
+          parseDate={str => new Date(str)}
+          onChange={this.handleDateChange}
+          value={[startDate, endDate]}
+        />
+      </div>
     )
   }
 
   render() {
-    if (this.isLoaded()) {
-      return (
-        <div>
+    if (!this.isLoaded()) {
+      return null
+    }
+
+    const { channels, shownSection, pageTitle, selectedChannel, showFilters } = this.state
+    const startDate = moment(moment.unix(this.state.startDate).toDate()).format('MMMM Do YYYY')
+    const endDate = moment(moment.unix(this.state.endDate).toDate()).format('MMMM Do YYYY')
+
+    return (
+      <Container>
+        <SidePanel>
+          <ul className={classnames(style.sideListList)}>
+            {sideList.map(item => (
+              <li
+                onClick={() => this.setState({ shownSection: item.key, pageTitle: item.label })}
+                key={item.key}
+                className={classnames(style.sideListItem, {
+                  [style.sideListItemSelected]: item.key === shownSection
+                })}
+              >
+                {item.label}
+              </li>
+            ))}
+          </ul>
+        </SidePanel>
+        <div className={style.mainWrapper}>
           <div className={style.header}>
-            Filter by&nbsp;
-            <HTMLSelect onChange={this.handleFilterChange} defaultValue="all">
-              {this.state.channels.map(channel => {
-                return <option value={channel}>{this.capitalize(channel)}</option>
-              })}
-            </HTMLSelect>
-            {this.renderDateInput()}
+            <h1 className={style.pageTitle}>{pageTitle}</h1>
+            <Button
+              onClick={() => this.setState(({ showFilters }: any) => ({ showFilters: !showFilters }))}
+              icon="filter"
+              className={style.filtersButton}
+            >
+              Filters
+            </Button>
           </div>
-          {this.renderAgentUsage()}
-          {this.renderEngagement()}
-          {this.renderUnderstanding()}
+          {showFilters && (
+            <div className={style.filtersWrapper}>
+              <label className={style.filtersInputWrapper}>
+                <span className={style.inputLabel}>Channel</span>
+                <HTMLSelect onChange={this.handleFilterChange} defaultValue="all">
+                  {channels.map(channel => {
+                    return (
+                      <option key={channel} value={channel}>
+                        {this.capitalize(channel)}
+                      </option>
+                    )
+                  })}
+                </HTMLSelect>
+              </label>
+              {this.renderDateInput()}
+            </div>
+          )}
+          <h2 className={classnames(style.appliedFilters, showFilters && style.filtersShowing)}>
```

Comment:
With classnames (i suggest using the shorthand "cx" as we do in other modules), you can use conditional styling this way:

```js
classnames(style.appliedFilters, { 
  [style.filtersShowing]: showFilters 
})
```

or 

```js
cx(style.appliedFilters, { [style.filtersShowing]: showFilters })
```